packages/graphql/src/translate/queryAST/ast/operations/composite/CompositeConnectionReadOperation.ts

```diff
@@ -138,4 +159,38 @@ export class CompositeConnectionReadOperation extends Operation {
             return [...nodeFields, ...edgeFields];
         });
     }
+
+    // NOTE: duplicate from ConnectionReadOperation
+    private transpileAggregation(context: QueryASTContext): {
```

Comment:
Not sure if it will be right approach the proposal I'm saying but:
Can we expose this method from the `aggregationField` and avoid duplicity between the `CompositeConnectionReadOperation` and the `ConnectionReadOperation`

Reply:
TBH, I don't think that is the best option. Sure, the code is the same, but it is more coincidental than anything, as transpilation for compositeConnection doesn't necessarily have to be the same, and I'm hesitant to couple the nested element of the tree. I'd rather have this duplicate code than break the encapsulation on those